Core/HLE/__sceAudio.cpp

```diff
@@ -264,7 +264,7 @@ u32 __AudioEnqueue(AudioChannel &chan, int chanNum, bool blocking) {
 		if (chan.format == PSP_AUDIO_FORMAT_STEREO) {
 			const u32 totalSamples = chan.sampleCount * 2;
 
-			s16_le *sampleData = (s16_le *) Memory::GetPointer(chan.sampleAddress);
+			s16 *sampleData = (s16 *) Memory::GetPointer(chan.sampleAddress);
```

Comment:
Well, this is stored in PSP RAM so isn't it LE?

-[Unknown]

Reply:
true, but for whatever reason, honoring that here will generate bad audio.
my guess is that all audio goes through decoders/FFMPEG, which all generate native endian audio, which is why the audio buffer here would be native endian "most" of the time. 
this is of course not ideal, but the correct way just happens to cost extra cpu time and require big changes to all decodes, for no benefit, so I'm not sure how to deal with this yet.
I also need to find a case where sampleData would actually contain little endian audio, to even be able to fix this.

Reply:
Most of the audio here will come from sceSas, sceAtrac, sceMpeg, or sceMp3.  But it's possible for games to generate the audio for sure - it's probably most common in homebrew, though (except in rare cases of games post-processing their audio - I can't remember any specific cases, though.)

-[Unknown]